From 584ad1b964239b00b487cc6a0686bf0ce1ccf7ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pablo=20Correa=20G=C3=B3mez?= <ablocorrea@hotmail.com>
Date: Mon, 17 Jan 2022 01:40:42 +0100
Subject: [PATCH] disable some non-desired preferences

The automatic updates can be dangerous, as applications are updated live,
and kernel or mkinitfs could be installed at any time. This includes
on very low battery or before user shuts-down the phone.

In addition, we only want free software apps. If technical users want
non-free apps, they can always undo the override
---
 src/gs-prefs-dialog.ui | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gs-prefs-dialog.ui b/src/gs-prefs-dialog.ui
index 415634981..940dfb43e 100644
--- a/src/gs-prefs-dialog.ui
+++ b/src/gs-prefs-dialog.ui
@@ -39,6 +39,7 @@
             </property>
             <child>
               <object class="AdwActionRow" id="automatic_updates_row">
+                <property name="sensitive">False</property>
                 <property name="title" translatable="yes">_Automatic</property>
                 <property name="subtitle" translatable="yes">Automatically check for and download updates</property>
                 <property name="activatable-widget">automatic_updates_radio</property>
@@ -86,6 +87,7 @@
             <property name="title" translatable="yes">Content Filters</property>
             <child>
               <object class="AdwSwitchRow" id="show_only_free_apps_row">
+                <property name="sensitive">False</property>
                 <property name="title" translatable="yes">Only Show _Free Software</property>
                 <property name="subtitle" translatable="yes">Hide proprietary software when browsing and searching</property>
                 <property name="use-underline">True</property>
-- 
2.46.1

